Seg fault processing a Deployment

Trying to helmify some manifests I have the following segmentation fault code:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1691f93]

goroutine 1 [running]:
github.com/arttor/helmify/pkg/processor/deployment.deployment.Process({}, {0x1906678, 0xc0000aed40}, 0xc00000e548)
	/home/runner/work/helmify/helmify/pkg/processor/deployment/deployment.go:71 +0x293
github.com/arttor/helmify/pkg/app.(*appContext).process(0xc0000e9d50, 0xc00000e548)
	/home/runner/work/helmify/helmify/pkg/app/context.go:75 +0x39c
github.com/arttor/helmify/pkg/app.(*appContext).CreateHelm(0xc0000e9d50, 0xc000308000)
	/home/runner/work/helmify/helmify/pkg/app/context.go:57 +0x252
github.com/arttor/helmify/pkg/app.Start({0x18f4d20, 0xc00000e010}, {{0x7ffeefbffa88, 0xc}, {0x17ee766, 0x1}, 0x0, 0x0})
	/home/runner/work/helmify/helmify/pkg/app/app.go:58 +0x737
main.main()
	/home/runner/work/helmify/helmify/cmd/helmify/main.go:21 +0x1b1
```

Someone had a similar situation?

--- a/pkg/processor/deployment/deployment.go
+++ b/pkg/processor/deployment/deployment.go
@@ -28,7 +28,9 @@ var deploymentGVC = schema.GroupVersionKind{
 var deploymentTempl, _ = template.New("deployment").Parse(
 	`{{- .Meta }}
 spec:
-  replicas: {{ .Replicas }}
+{{- if .Replicas }}
+{{ .Replicas }}
+{{- end }}
   selector:
 {{ .Selector }}
   template:
@@ -68,7 +70,7 @@ func (d deployment) Process(appMeta helmify.AppMetadata, obj *unstructured.Unstr
 	values := helmify.Values{}
 
 	name := appMeta.TrimName(obj.GetName())
-	replicas, err := values.Add(int64(*depl.Spec.Replicas), name, "replicas")
+	replicas, err := processReplicas(name, &depl, &values)
 	if err != nil {
 		return true, nil, err
 	}
@@ -162,6 +164,22 @@ func (d deployment) Process(appMeta helmify.AppMetadata, obj *unstructured.Unstr
 			Spec:           spec,
 		},
 	}, nil
+}
+
+func processReplicas(name string, deployment *appsv1.Deployment, values *helmify.Values) (string, error) {
+	if deployment.Spec.Replicas == nil {
+		return "", nil
+	}
+	replicasTpl, err := values.Add(int64(*deployment.Spec.Replicas), name, "replicas")
+	if err != nil {
+		return "", err
+	}
+	replicas, err := yamlformat.Marshal(map[string]interface{}{"replicas": replicasTpl}, 2)
+	if err != nil {
+		return "", err
+	}
+	replicas = strings.ReplaceAll(replicas, "'", "")
+	return replicas, nil
 }
 
 func processPodSpec(name string, appMeta helmify.AppMetadata, pod *corev1.PodSpec) (helmify.Values, error) {

